server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java

```diff
@@ -402,6 +410,13 @@ Statistics currentPageStatistics() {
     return firstPageReader.getStatistics();
   }
 
+  boolean currentPageModified() {
+    if (firstPageReader == null) {
```

Comment:
if firstPageReader is null, throw exception

Reply:
fixed